cedar-policy/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `wasm` Cargo feature for targeting Wasm
 - `Entity::into_inner` (resolving #636)
 - `Entities::into_iter` (resolving #680)
+- Adds an EST representation for Policy Sets (#783, resolving #549)
```

Comment:
nit: maybe list the specific methods being added.  People don't necessarily know what "EST" is, that is internal terminology